server/embed/config_test.go

```diff
@@ -843,7 +843,7 @@ func TestLogRotation(t *testing.T) {
 			logOutputs:        []string{"/tmp/path"},
 			logRotationConfig: `{"maxsize": true}`,
 			wantErr:           true,
-			wantErrMsg:        errors.New("invalid log rotation config: json: cannot unmarshal bool into Go struct field logRotationConfig.maxsize of type int"),
+			wantErrMsg:        errors.New("invalid log rotation config: json: cannot unmarshal bool into Go struct field logRotationConfig.Logger.maxsize of type int"),
```

Comment:
question: is this a side effect to bump the go version?

Reply:
Yes, the error message is different now, it also shows the embedded structure.

Reply:
I added more context in 31650ab0c8df43af05fc4c13b48ffee59271eec7's commit message:

>     The embedded structure is now also part of the log message. The failure
>     from the tests is the following:
>    
>     --- FAIL: TestLogRotation (0.00s)
>         --- FAIL: TestLogRotation/invalid_logger_config (0.00s)
>             config_test.go:868: test "invalid logger config", expected
>             error: invalid log rotation config: json: cannot unmarshal
>                    bool into Go struct field logRotationConfig.maxsize
>                    of type int,
>             got: invalid log rotation config: json: cannot unmarshal
>                  bool into Go struct field logRotationConfig.Logger.maxsize
>                  of type int
>     FAIL